src/SelfService/Application/PlatformDataApiRequesterService.cs

```diff
@@ -11,7 +11,7 @@ public class PlatformDataApiRequesterService : IPlatformDataApiRequesterService
 {
     #region ResponseObjects
 
-    private class PlatformDataApiTimeSeries
+    public class PlatformDataApiTimeSeries
```

Comment:
It's always a bit sad when testing forces private things to become public :(
Oh well 🤷‍♂️